ebean-ddl-generator/src/main/java/io/ebeaninternal/dbmigration/ddlgeneration/platform/BaseTableDdl.java

```diff
@@ -803,28 +776,20 @@ protected void regenerateHistoryTriggers(String baseTableName, HistoryTableUpdat
    * This is mysql specific - alter all the base attributes of the column together.
    * Will be called, if there is a type, dbdefault or notnull change.
    */
-  protected void alterColumnBaseAttributes(DdlWrite writer, AlterColumn alter) {
-    String ddl = platformDdl.alterColumnBaseAttributes(alter);
-    if (hasValue(ddl)) {
-      writer.apply().appendStatement(ddl);
-
-      if (isTrue(alter.isWithHistory()) && alter.getType() != null && historySupport == HistorySupport.TRIGGER_BASED) {
-        // mysql and sql server column type change allowing nulls in the history table column
-        regenerateHistoryTriggers(alter.getTableName(), HistoryTableUpdate.Change.ALTER, alter.getColumnName());
-        AlterColumn alterHistoryColumn = new AlterColumn();
-        alterHistoryColumn.setTableName(historyTable(alter.getTableName()));
-        alterHistoryColumn.setColumnName(alter.getColumnName());
-        alterHistoryColumn.setType(alter.getType());
-        String histColumnDdl = platformDdl.alterColumnBaseAttributes(alterHistoryColumn);
-
-        // write the apply to history table
-        writer.apply().appendStatement(histColumnDdl);
-      }
-    }
-  }
+  protected void alterColumn(DdlWrite writer, AlterColumn alter) {
+    platformDdl.alterColumn(writer, alter);
+
+    if (isTrue(alter.isWithHistory()) && alter.getType() != null && historySupport == HistorySupport.TRIGGER_BASED) {
```

Comment:
CHECKME: 
testing for `alter.getType() != null` seems OK here (we do not need to change default/nullness on history table) but we might need `alterHistoryColumn.setCurrentNotNull` and `alterHistoryColumn.setCurrentDefault` in order NOT to drop existing ones.
We might also handle the case when a nonnull column becomes null